modules/030-cloud-provider-yandex/docs/README.md

```diff
@@ -11,3 +11,241 @@ The `cloud-provider-yandex` module:
 - Provisions disks in Yandex Cloud using the `CSI storage` component.
 - Registers with the [node-manager](../../modules/040-node-manager/) module so that [YandexInstanceClasses](cr.html#yandexinstanceclass) can be used when creating the [NodeGroup](../../modules/040-node-manager/cr.html#nodegroup).
 - Enables the necessary CNI plugin (using the [simple bridge](../../modules/035-cni-simple-bridge/)).
+
+## Yandex Cloud integration
+
+### Configuring security groups
+
+When creating a [cloud network](https://cloud.yandex.com/en/docs/vpc/concepts/network#network), Yandex Cloud creates a default [security group](https://cloud.yandex.com/en/docs/vpc/concepts/security-groups) for all networks, including the Deckhouse Kubernetes Platform cluster network. The default security group contains rules that allow for any traffic to pass in any direction (inbound and outbound) and applies to all subnets within the cloud network, unless an object (VM interface) is explicitly assigned to a different security group. You can change the default security group rules if you need to control traffic in your cluster.
+
+This section provides general guidelines for setting up a security group. Incorrect configuration of security groups may affect the performance of the cluster. Please consult [security group usage details](https://cloud.yandex.com/en/docs/vpc/concepts/security-groups#security-groups-notes) in Yandex Cloud before using it in production environments.
+
+{% alert level="danger" %}
+Do not delete the rule that allows for traffic to pass in any direction before finishing configuring all the other rules for the security group. Doing so may disrupt the performance of the cluster.
+{% endalert %}
+
+1. Find out in which cloud network the Deckhouse Kubernetes Platform cluster is running.
+
+   The network name matches the `prefix` field of the [ClusterConfiguration] resource(../../installing/configuration.html#clusterconfiguration). It can be retrieved using the following command:
+
```

Comment:
```suggestion
   The network name matches the `prefix` field of the [ClusterConfiguration](../../installing/configuration.html#clusterconfiguration) resource. It can be retrieved using the following command:

```